rpa-bot/1.0/modules/ROOT/edits/configuration.adoc

```diff
@@ -0,0 +1,137 @@
+* *RPA Manager to connect*
++
+The URL address of the RPA Manager instance the bot connects to. There is no default setting.
+* *Use HTTPS as protocol layer*
++
+Specifies whether the bot connects with RPA Manager via HTTP or HTTPS.
+* *API Key*
++
+The API Key entered during the installation of the robot. You can change this setting.
+* *Alive interval*
++
+Time interval in milliseconds after which the bot connects to the RPA Manager to show that it is still alive.
+* *Check interval*
++
+Time interval in milliseconds after which the bot connects to RPA Manager and checks for provided packages to collect (such as Scheduling Tasks or Activities).
+* *Cleanup workflows on start*
++
+Select this option to delete the workflows that are no longer assigned to a configuration but are still stored locally in the bot.
+
+[[proxy-settings]]
+== Proxy Settings
+
+image:bot-configurator-proxy.png[The RPA Bot configurator application showing the Proxy settings panel, 50%, 50%]
+
+* *Proxy*
++
+The bot and RPA Manager communicate through the HTTP protocol at Port 80. If you
+connect to the internet through a proxy server, enter the proxy address in this field
++
+Format: `+http://<Serveraddress>:<Port>+`, for example: `+http://myproxy.de:881+`.
+
+[[secure-session]]
+== Secure Session
+
+Enable the bot to use Secure Sessions by entering credentials the bot uses to login to the computer and to open a hidden session. If you don't add credentials here, the bot can only use a running desktop session.
+
+image:bot-configurator-secure-session.png[The RPA Bot configurator application showing the Secure Session settings panel, 50%, 50%]
+
+* *Default Secure Session User*
++
+If you want the bot to use Secure Sessions, enter the name of the user the bot uses to login to the computer and to run a hidden session. If the environment of the computer the bot is installed on is embedded in a domain and you want to use a local user, write the user name as follows: `.\username`.
+* *Activate certificate based login on SSL enabled terminal servers*
++
+Enable this option if you are installing the bot on a terminal server that only allows SSL protected RDP connections.
+* *Handle “Always prompt client for password” option*
++
+There is a Group Policy called `fPromptForPassword`. If this policy is enabled, the password has to be entered a second time, when the RDP session is established. If this is the case, enable this option. Also choose the correct form of dialog. On some machines the second password entry is requested by a separate window in form of
+a popup dialog. And on other machines it is an embedded screen like the normal windows login.
+//To determine if the option as to be used and which form of dialog needs to be configured, it is a good idea to execute the steps described in chapter Functional test.
```

Comment:
confirm this should be commented out. Also, we don't refer to docs as "chapters"